### DISCUSSION TITLE
YI-Outdoor Camera 1080p RFUS YHS.3017

### DISCUSSION THREAD

**Discussion by Horst-Kno-61** [roleoroleo/yi-hack-Allwinner-v2#506] [Q&A]
Hi,
I have a YI Outdoor Camera 1080p RFUS
YHS.3017 ; SN: RH30GA......; Firmware: 9.0.26.08_202203181055.
Unfortunately I couldn't find a suitable Flash.
Tried everything, unfortunately no success.
I'm grateful for any help!

**Comment by roleoroleo** [comment]
h30ga should work.

**Comment by Horst-Kno-61** [comment]
I thought so too.
But I don't get a backup nor that
Playing the hack.
I worked with 2 different memory cards (8-16 GB)
and different formatting methods.

**Reply by roleoroleo** [reply]
Is Factory folder renamed to Factory.done?

**Comment by Horst-Kno-61** [comment]
So far I haven't tried renaming
But it's useless.
When backing up, the camera ignores the directory and starts.
The camera does not start when hacked.

**Comment by Horst-Kno-61** [comment]
Once reset the camera
and the camera is unlocked.
Backup and hack work.
Thanks for the help!!

**Comment by roleoroleo** [comment]
The cam doesn't read the sd card.
Incompatibility with the card or the file system.
Or hardware failure.

**Reply by TZSENS** [reply]
reflash today for my outdoor 1080 RFUS . it works, fantastic. did a donation to show my appriciation.

**Reply by roleoroleo** [reply]
Thank you so much for your donation.
